### PR TITLE
Class space

### PR DESCRIPTION
**Pull request by cristianfchq**
Hello team here is the class class-space for review You can view, comment on, or merge this pull request.

### CODE REVIEW

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.at13.spaceinvaders;
+
+public class Space{
+
+    private static final int SPACE_SIZE_HIGHT=50;
+    private static final int SPACE_SIZE_WIDTH=100;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java:6]: Space is missing between "=", in line 5 and 6.

File: src/test/java/org/fundationjala/at13/spaceinvaders/MenuTest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package org.fundationjala.at13.spaceinvaders;
+import org.fundacionjala.at13.spaceinvaders.Menu;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+
+public class MenuTest {
+    
+    @Test
+    public void itDisplaySpaceByOptionOne() {
```
Comment (Paola-Xaq) [inline src/test/java/org/fundationjala/at13/spaceinvaders/MenuTest.java:10]: This method said that "Displays the Space by option 1". But in Lin 17 the method said that Display by option 0.

File: src/test/java/org/fundationjala/at13/spaceinvaders/MenuTest.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        Menu menu = new Menu();
+        boolean result = menu.menuOptions(1);
+        assertTrue(result);
+    }
+
+    @Test
+    public void itDisplaySpaceByOptionZero() {
```
Comment (Paola-Xaq) [inline src/test/java/org/fundationjala/at13/spaceinvaders/MenuTest.java:17]: Method name is Wrong this option is for Exit the Menu.

**Review by Paola-Xaq — CHANGES_REQUESTED** [review]
Hi there,
I just review the pull request, here are my observations.

Regards,
Pao

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java (hunk 1 above)
Comment (cristianfchq) [reply src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java:6]: ok, sorry!!!

**Pull request merged**